packages/smart-router/evm/getStableSwapPairs.ts

```diff
@@ -0,0 +1,23 @@
+import { ChainId, CurrencyAmount, Pair } from '@pancakeswap/sdk'
+import { getStableConfig } from '@pancakeswap/farms/constants'
```

Comment:
shouldn't rely on farms config 🤔 
that might be a case that stableswap pair is not in farms
let's add a constants for stableswap

Reply:
🤔 I thought we have farms for all pairs we supported. Let me add it's own config.

Reply:
Do we have a lp config that maintains all token pairs that we support?